src/use_case_classification.Rmd

```diff
@@ -0,0 +1,147 @@
+library(BBmisc)
+```
+
+
+```{r}
+data = titanic_train
+head(data)
+```
+
+Our aim - as mentioned before - is to predict which kind of people would have survided.
+
+Therefore we will work off the following steps:
+
+* preprocessing, [here](http://mlr-org.github.io/mlr-tutorial/devel/html/preproc/index.html) and [here](http://mlr-org.github.io/mlr-tutorial/devel/html/impute/index.html)
+* create a task,[here](http://mlr-org.github.io/mlr-tutorial/devel/html/task/index.html)
+* provide a learner, [here](http://mlr-org.github.io/mlr-tutorial/devel/html/learner/index.html)
+* train the model, [here](http://mlr-org.github.io/mlr-tutorial/devel/html/train/index.html)
+* predict the survival chance, [here](http://mlr-org.github.io/mlr-tutorial/devel/html/predict/index.html)
+* validate the model,[here](http://mlr-org.github.io/mlr-tutorial/devel/html/performance/index.html) and [here](http://mlr-org.github.io/mlr-tutorial/devel/html/resample/index.html)
+
+#### Preprocessing
+
+The data set is corrected regarding their data types.
+
+```{r}
+data[, c("Survived", "Pclass", "Sex", "SibSp", "Embarked")] = lapply(data[,  c("Survived", "Pclass", "Sex", "SibSp", "Embarked")], as.factor)
+```
+
+Next, unuseful columns will be dropped.
+
+```{r}
+data = dropNamed(data, c("Cabin","PassengerId", "Ticket", "Name"))
+```
+
+And missing values will be imputed, in this case Age and Fare.
+
+```{r}
+data$Embarked[data$Embarked == ""] = NA
+data$Embarked = droplevels(data$Embarked)
+data = impute(data, cols = list(Age = imputeMedian(), Fare = imputeMedian(), Embarked = imputeMode()))
+data = data$data
+```
+
+### Create a task
+
+In the "task" the data set and the target column is specified. People who survived are labelled with "1".
+
+```{r}
+task = makeClassifTask(data = data, target = "Survived", positive = "1")
+```
+
+### Define a learner
+
+A classification learner is selected. You can find an overview of all learners [here](http://mlr-org.github.io/mlr-tutorial/devel/html/integrated_learners/index.html)
+
+```{r}
+lrn = makeLearner("classif.randomForest", predict.type = "prob")
+```
+
+### Fit the model
+
+To fit the model - and afterwards predict - the data set is split into a training and a test data set.
+
+```{r}
+n = getTaskSize(task)
+trainSet = seq(1, n, by = 2)
+testSet = seq(2, n, by = 2)
+```
+
+```{r}
+mod = train(learner = lrn, task = task, subset = trainSet)
+```
+
+### Predict
+
+Predicting the target values for new observations is implemented the same way as most of the other predict methods in R. In general, all you need to do is call predict on the object returned by train and pass the data you want predictions for.
+
+```{r}
+pred = predict(mod, task, subset = testSet)
+```
+
+The quality of the predictions of a model in mlr can be assessed with respect to a number of different performance measures. In order to calculate the performance measures, call performance on the object returned by predict and specify the desired performance measures.
```

Comment:
* In the first sentence in l. 93 please link to the performance measure table in the tutorial: `[number of different performance measures](measures.md)`.
* Again I think the 2nd sentence is too technical. For the use cases I wouldn't explain function APIs (that's what the tutorial and manual pages are for). Just say concisely what you are doing (in this instance, calculate accuracy as well as false and true positive rates) and the reader will make the connection to the `performance` call.
* You are doing a lot more things than the `performance` call in the code chunk below. Please give more info about that.